test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -282,6 +289,42 @@ def test_emacs(image):
     _run_cmd_on_container(container_name, ctx, "emacs -version")
 
 
+@pytest.mark.model("N/A")
+@pytest.mark.integration("sagemaker python sdk")
+def test_sm_pysdk_version(image):
+    """
+    Simply verify that we have sagemaker > 2 in the python sdk
+
+    :param image: ECR image URI
+    """
+    ctx = Context()
+    container_name = _get_container_name("sm_pysdk", image)
+    _start_container(container_name, image, ctx)
+
+    framework, framework_version = get_framework_and_version_from_tag(image)
+    python_version = re.search(r"py\d+", image).group()
+    job_type = get_job_type_from_image(image)
+
+    if framework == "tensorflow":
+        if is_tf_version("2", image):
+            framework = "tensorflow2"
+        elif is_tf_version("1", image):
+            framework = "tensorflow1"
+
+    sm_pysdk_v2 = {"tensorflow2": {"training": {"py37": "2.3.0"}}, "tensorflow1": {"training": {"py36": "1.15.0"}}}
+
+    sm_version = _run_cmd_on_container(
+        container_name, ctx, "import sagemaker; print(sagemaker.__version__)", warn=True, executable="python"
+    ).stdout.strip()
+
+    whitelisted_fw_version = sm_pysdk_v2.get(framework, {}).get(job_type, {}).get(python_version)
+    if whitelisted_fw_version:
+        if version.parse(framework_version) > version.parse(whitelisted_fw_version):
+            assert version.parse(sm_version) > version.parse(
+                "2.0.0"
+            ), f"Sagemaker version should be greater than 2. Found version: {sm_version}"
```

Comment:
`Version(X)` is functionally equivalent and syntactically better compared to `parse(X)`.

Also, `parse("2.0.0")` is unnecessarily verbose, and can just be `parse("2")`, or `Version("2")`.